src/main/java/uk/gov/digital/ho/hocs/casework/client/infoclient/InfoClient.java

```diff
@@ -145,13 +145,18 @@ public TeamDto getTeamByStageAndText(String stageType, String text) {
         return response;
     }
 
-    @Cacheable(value = "InfoClientGetCaseDeadline", unless = "#result == null", key = "{#caseType, #received.toString(), #days.toString() }")
     public LocalDate getCaseDeadline(String caseType, LocalDate received, int days) {
         LocalDate response = restHelper.get(serviceBaseURL, String.format("/caseType/%s/deadline?received=%s&days=%s", caseType, received, days), LocalDate.class);
         log.info("Got {} as deadline for CaseType {} and Date {} and Days {}", response.toString(), caseType, received, days, value(EVENT, INFO_CLIENT_GET_CASE_DEADLINE_SUCCESS));
         return response;
     }
 
+    public LocalDate getCaseDeadline(String caseType, LocalDate received, int days, int extensionDays) {
```

Comment:
Should we be changing the name on this method, given the different usecase for it? Appreciate that it should have been picked up on the initial review but maybe a chance to correct?